static/locales/de.json

```diff
@@ -1457,14 +1744,27 @@
   "Wigglytuff": "Knuddeluff",
   "Wild Charge": "Stromstoß",
   "Will-O-Wisp": "Irrlicht",
+  "Wimpod": "Reißlaus",
+  "Windy": "Windig",
   "Wing Attack": "Flügelschlag",
   "Wingull": "Wingull",
+  "Winter": "Winterform",
```

Comment:
Wrong translation of "Winter"

Reply:
https://pokefans.net/spiele/pokemon-go/pokemon/sesokitz-winterform
see autum reply